wire-library/wire-gson-support/gradle.properties

```diff
@@ -1,3 +0,0 @@
-POM_ARTIFACT_ID=wire-gson-support
-POM_NAME=Wire
```

Comment:
Out of curiosity, why not keep the properties files and avoid the explicit pom configuration in the base plugin dsl?

Reply:
I personally prefer the safety of the base DSL. IntelliJ automatically completes the available fields vs. properties files where I’m always anxious I’ve got a silent typo.